packages/grpc-js/src/resolver-dns.ts

```diff
@@ -301,9 +335,9 @@ class DnsResolver implements Resolver {
   }
 
   destroy() {
-    /* Do nothing. There is not a practical way to cancel in-flight DNS
```

Comment:
This comment is interesting because in the meantime, I learned that there is, in fact, a way to cancel in-flight DNS requests in libuv. Funny they didn't necessarily expose this to the upper JavaScript?

http://docs.libuv.org/en/v1.x/request.html

`uv_cancel(uv_req*);`

Reply:
If they wanted to do that they'd have to return some kind of handle object from `dns.lookup` that would exist solely for the unusual edge case that you want to cancel a DNS request.